charts/kspm-collector/templates/deployment.yaml

```
@@ -16,11 +20,11 @@ spec:
     metadata:
       labels:
         app.kubernetes.io/name: {{ include "kspmCollector.name" . }}
+{{ include "kspmCollector.labels" . | indent 8 }}
```

Can I suggest to use
```suggestion
        {{ include "kspmCollector.labels" . | nindent 8 }}
```
to make the document format more easy to understand? WDYT?